types/pdfkit/index.d.ts

```diff
@@ -3,7 +3,7 @@
 declare namespace PDFKit {
     interface PDFGradient {
         new(document: any): PDFGradient;
-        stop(pos: number, color?: string | PDFKit.PDFGradient, opacity?: number): PDFGradient;
+        stop(pos: number, color?: string | number[] | PDFKit.PDFGradient, opacity?: number): PDFGradient;
```

Comment:
```suggestion
        stop(pos: number, color?: ColorValue, opacity?: number): PDFGradient;
```
 since the docs say the valid colors should be identical to the `fillColor` arguments:

 https://pdfkit.org/docs/vector.html#gradients
 https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c80c5aaf673fcdd13e07c1f2d92852ad7fe652a2/types/pdfkit/index.d.ts#L348